eregs_core/management/commands/cache_content.py

```diff
@@ -0,0 +1,59 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.core.exceptions import ObjectDoesNotExist
+from eregs_core.models import *
+from eregs_core.utils import xml_to_json
+from lxml import etree
+from itertools import product, chain
+
+import os
+import json
+import requests
+
+
+class Command(BaseCommand):
+
+    help = 'Cache the views of the specified regulation.'
+
+    def add_arguments(self, parser):
+        parser.add_argument('regs', nargs='*', type=str)
+
+    def handle(self, *args, **options):
+        regs_to_cache = options['regs']
+        preambles = Preamble.objects.filter(tag='preamble').exclude(reg_version__version=None)
+        base = 'http://localhost:8000'
```

Comment:
How is this intended to be used in production? This code requires programmatic access to the database but also hits the pages via `localhost:8000`, which wouldn't necessarily be active. What about just programmatically rendering the views via code instead, and bypassing `requests`/the external server altogether?